### DISCUSSION TITLE
Mostrar en la pagina lo que esta en la consola (Pokedex)

### DISCUSSION THREAD

**Discussion by Juan-2003** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#470] [Q&A] [answered]
Ya extrai el tipo de Pokémon que es a partir del API, ya me aparece la consola si es de tipo  fuergo, agua, electrico, etc. Pero no he hallado la manera que se muestre en pantalla. Trate de hacerlo cambiando el contenido de un "Div" de HTML, pero nada.
¿Cómo lo han hecho ustedes?

**Comment by Ingrzs** [comment]
Con el inner html en la práctica 3 de JavaScript esta el paso para pasarlo de la api a html

**Comment by doguedogue** [comment, marked as answer]
Si en tu HTML tienes un input para tu tipo, este tiene un atributo value que es el valor que se tiene adentro
`<input type="text" id="tipo" value="">`

En el JS cuando obtienes el valor que obtuviste con la API lo insertas así:
`const inputfield = document.getElementById("tipo"); inputfield.value = tipo;`

**Reply by LuisAlias** [reply]
Muchas Gracias Amigo, me fue de mucha ayuda, y ya pude visualizar lo de la API en la pagina HTML. Gracias, Gracias.
